client/CMakeLists.txt

```diff
@@ -58,6 +51,10 @@ set(trillek-client_SOURCES ${trillek-client_SOURCES} PARENT_SCOPE) # so tests ca
 set(OSX_LIBRARIES "")
 if (APPLE)
 	set(OSX_LIBRARIES "/usr/lib/libobjc.dylib")
+	
+	# Rather wonderfully, Apple decided to mark every single fucking OpenGL function
+	# as deprecated because they have 'deprecated' OpenGL. Brilliant!
+    add_definitions(-DGL_SILENCE_DEPRECATION)
```

Comment:
I like that you kept the comment intact XD

Reply:
Well comments like that are important, so we don't scratch our heads later like why did we need this define?